CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+ * [`changed`]  Return types are now `int16_t` instead of `int8_t` (in line with
+                other embedded drivers).
+ * [`changed`]  Functions are now dedicated per sensor (e.g. `sht3x_probe()`
+                instead of `sht_probe()`)
+ * [`added`]    New return values for wakeup/sleep failed
+                (`STATUS_WAKEUP_FAILED`, `STATUS_SLEEP_FAILED`)
+ * [`fixed`]    Fix SHTC3 issues when calling `probe()` multiple times
+ * [`changed`]  Move the i2c init call out of `probe()` and into the example
+ * [`changed`]  Rename `git_version.[ch]` to `sht_git_version.[ch]`
+ * [`changed`]  Dedicated sht commands per sensor type
```

Comment:
Isn't this a duplicate of the" Functions are now dedicated per sensor" entry?

Reply:
yup, I probably thought it's twice as important :roll_eyes: